tasks/minimize_access.yml

```diff
@@ -18,9 +18,16 @@
   when: item.stat.isdir
   with_items: "{{ minimize_access_directories.results }}"
 
-- name: change shadow ownership to root and mode to 0600 | DTAG SEC Req 3.21-7
+- name: change shadow ownership to {{ os_shadow_perms.owner }}.{{ os_shadow_perms.group }}  and mode to {{ os_shadow_perms.mode }} | os-03
   file: dest='/etc/shadow' owner={{ os_shadow_perms.owner }} group={{ os_shadow_perms.group }} mode={{ os_shadow_perms.mode }}
```

Comment:
Seems like you duplicated this task above (new task is below)  and changed it to proper YAML syntax. Sounds like you can remove the above task now?